match/lib/match/nuke.rb

```diff
@@ -173,6 +178,77 @@ def prepare_list
       self.files = certs + keys + profiles
     end
 
+    def filter_by_cert
+      # Force will continue to revoke and delete all certificates and profiles
+      return if self.params[:force]
+      return if self.certs.count < 2
```

Comment:
i think, we should early return if this func will be running on CI, because users cant input/select specific certificates on CI? 🤔

Reply:
Good point 👍. 
Would `return unless UI.interactive?` do the job? 
If it does, other possibility would be check if we're in interactive mode in `run`
```ruby
prepare_list
filter_by_cert if UI.interactive?
print_tables
```